app/packages/plugin.js

```diff
@@ -133,13 +133,6 @@ class Plugin extends Package {
     this.blocksById[output.id] = output
   }
 
-  respondsTo (inputText) {
-    if (!this.loaded || !this.activeState) { return }
-    return this.inputs.find((input) => {
```

Comment:
`this.activeState` is no longer being checked, so scoped blocks aren't scoped correctly. For example the clipboard plugin.